src/test/scala/firrtlTests/options/PhaseManagerSpec.scala

```diff
@@ -392,7 +392,7 @@ class PhaseManagerSpec extends AnyFlatSpec with Matchers {
 
     /** Convert a Graphviz file to PNG using */
     def maybeToPng(f: File): Unit = try {
-      s"dot -Tpng -O ${f}" !
```

Comment:
no need to change here.

Reply:
This is the same postfix issue as with the `toSeq` above.

Reply:
You can fix postfix by just adding a dot

Reply:
ie.
```scala
    def maybeToPng(f: File): Unit = try {
      s"dot -Tpng -O ${f}".!
      ...
```

Reply:
That is exactly what I did.

Reply:
(Somehow github is showing only one line of the change, but if you click on the file name it will show you the red and the green line).

Reply:
Lol got it, thanks